tools/pontoon/translate.md

```diff
@@ -0,0 +1,105 @@
+
+![Sidebar after suggestion](/assets/images/pontoon/translate/sidebar_suggest.png)
+
+A notification will appear at the top of the window, indicating that your translation has been saved.
+
+Note that a suggested translation exists only in the Pontoon database, not in the localized file or translation memories.
+
+#### Manually switch to suggestion mode
+
+Even if a translator has permissions to add translations directly, submitting suggestions to be reviewed helps ensure quality. To manually switch to Suggestion Mode, click the Settings gear to the lower-left side of the editing space and click the icon to the left of `Make Suggestions` so that it becomes a green circle. The `SUGGEST` button should then appear to the lower-right side of the editing space.
+
+![Activate make suggestions](/assets/images/pontoon/translate/activate_suggest.png)
+
+A notification will appear at the top of the window, indicating that `Make Suggestions` has been enabled.
+
+To switch to Review and Direct Translation Mode, go to the settings menu by clicking on the gear and turn off the `Make Suggestions` option. The green `SAVE` button should appear to the lower-right side of the editing space.
+
+### Review a suggestion
+
+To review a suggestion, search for suggested strings or click on a suggested string in the sidebar.
+
+If the suggested string is an acceptable translation, either click on the `SAVE` button to the lower-right side of the editing space, or click on the blue circle to the right of the string. This will transform the suggested string into an approved translation, save the string to the database and to the localized file on the next sync operation.
+
+If the suggested string is **not** an acceptable translation, turn on `Make Suggestions` in settings. Once the `SUGGEST` button appears on the lower-right side of the editing space, edit the suggested string as appropriate and click `SUGGEST`. This will add your translation as a new suggested string in the Pontoon database but will not save the string in the localized file. To see the old and new suggestions, click on the history tab below the editing space. To learn about the history tab and other resources, check the [Translation resources documentation](resources.md)
+
+![History Tab](/assets/images/pontoon/translate/history_tab.png)
+
+If there are multiple suggestions for one string, it’s possible to use the `SHOW DIFF` link displayed on the right side of the suggestion: it will compare the selected suggestion to the current content of the editing area (approved translation, or most recent suggestion), highlighting in green all added text, in red all removed text. `HIDE DIFF` will display the original string.
```

Comment:
This paragraph is copied from another document. Maybe we can link there instead?

If not, the same comment applies:
Diff compares the selected suggestion to the first entry. The "current content of the editing area" could be misleading, in case the user changes the content.